docs/source/command-reference.mdx

```diff
@@ -85,7 +85,7 @@ Specifying either the HTTP port or address (or both) will enable the Streamable
 
 ### Mapping rover dev options
 
-You can use the [`rover dev`](/rover/commands/dev) command of Rover CLI v0.31 or later to run an Apollo MCP Server instance for local development. 
+You can use the [`rover dev`](/rover/commands/dev) command of Rover CLI v0.32 or later to run an Apollo MCP Server instance for local development. 
```

Comment:
0.32 is not technically required but it is recommended since we recommend Streamable HTTP. Are you ok with the change @pubmodmatt